### DISCUSSION TITLE
What `docstyle` do we use for `sphinx`?

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [pyvista/pyvista#2906] [Q&A] [answered]
Just want to verify, do we use the [numpy docstyle](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy) in `pyvista` when making docstrings?

**Comment by tkoyama010** [comment, marked as answer]
Yes, it is. But it is not explicitly stated in the [Contribution](https://dev.pyvista.org/extras/developer_notes.html) section. We will need to add it.

**Reply by tkoyama010** [reply]
See https://github.com/pyvista/pyvista/issues/2913
